exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/http/logs/OtlpHttpLogRecordExporter.java

```diff
@@ -84,4 +84,9 @@ public CompletableResultCode flush() {
   public CompletableResultCode shutdown() {
     return delegate.shutdown();
   }
+
+  @Override
+  public String toString() {
+    return "OtlpHttpLogRecordExporter{builder=" + builder + "}";
```

Comment:
I wonder if we should not tag this as `builder` . I suspect that will be very confusing to have a concrete exporter report itself as having a builder embedded in it.  🤔

Reply:
Yeah I considered that myself. I can switch it.